vit-servicing-station-tests/src/common/data/mod.rs

```diff
@@ -0,0 +1,60 @@
+use chrono::offset::Utc;
+use vit_servicing_station_lib::{
+    db::models::{
+        api_tokens::APITokenData,
+        proposals::{Category, Proposal, Proposer},
+        vote_options::VoteOptions,
+    },
+    v0::api_token::APIToken,
+};
+
+pub struct TestGen;
```

Comment:
There is no need to encapsulate this in a struct. Usually, you can have the same behavior simply with a `mod` (like namespacing) and public functions.